lib/firmenwissen/session.rb

```diff
@@ -0,0 +1,26 @@
+module Firmenwissen
+  module Session
+    module_function
+
+    def cookies
+      @cookies ||= {}
+    end
+
+    def update_from_set_cookie_headers(set_cookie_headers)
+      return if set_cookie_headers.nil?
+
+      set_cookie_headers.each do |header|
+        name_and_value, _other_attrs = header.split(/;\s?/)
+        name, value = name_and_value.split('=', 2)
+
+        next if name.empty? || value.nil? || value.empty?
```

Comment:
Are empty cookies the same as not-set cookies?

Reply:
I am not sure what you refer to with `not-set` – should we have a brief OTR chat on that?